.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Type of change
+
+<!-- Add an x to the categories that apply -->
+
+- [ ] Documentation change
+- [ ] Bug fix
+
+## Summary of change
+
+<!-- Please include a brief summary on the changes made in this PR. You should also include any screenshots or videos when applicable -->
```

Comment:
"of" instead of "on"?

Reply:
Ohh yeah, that makes sense. 
Will change that now 👍